### PR TITLE
[RHACS] ROX15870 Documenting ACS support for OCP 4.13 + RHEL 9.2

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.0

Issue:
https://issues.redhat.com/browse/ROX-15870

Link to docs preview:
https://57528--docspreview.netlify.app/openshift-acs/latest/installing/acs-installation-platforms.html#installation-methods-for-different-architectures_acs-installation-platforms

SME review:
- [x] SME has approved this change.

Additional information:
Merge into `rhacs-docs` and cherry-pick into:
- `rhacs-docs-4.0`

### CODE REVIEW

**Comment by agantony** [comment]
SME review needed: Robby Cochran

**Review by robbycochran — APPROVED** [review]

**Comment by agantony** [comment]
Peer review needed: Kerry

File: modules/installation-methods-for-different-architectures.adoc [hunk 1]
```diff
@@ -20,11 +20,11 @@ a|Operator (preferred), Helm charts, or roxctl CLI (not recommended)
 
 | ppc64le ({ibm-power})
 |No
-|Yes ({ocp} version 4.12 only)
+|Yes ({ocp} version 4.12 and newer)
```
Comment (kcarmichael08) [inline modules/installation-methods-for-different-architectures.adoc:23]: Use "later" for subsequent software versions per IBM Style.

Comment (kcarmichael08) [inline modules/installation-methods-for-different-architectures.adoc:23]:
```suggestion
|Yes ({ocp} version 4.12 and later)
```

File: modules/installation-methods-for-different-architectures.adoc [hunk 2]
```diff
@@ -20,11 +20,11 @@ a|Operator (preferred), Helm charts, or roxctl CLI (not recommended)
...
 .2+a|Operator is the only supported install method.
 
 | s390x ({ibm-zsystems} and {ibm-linuxone})
 |No
-|Yes ({ocp} versions 4.10 and 4.12 only)
+|Yes ({ocp} versions 4.10, 4.12 and newer)
```
Comment (kcarmichael08) [inline modules/installation-methods-for-different-architectures.adoc:28]: same comment as above

**Review by kcarmichael08 — COMMENTED** [review]
One small issue; otherwise, lgtm!

**Comment by agantony** [comment]
Merge review needed: Gaurav

File: modules/installation-methods-for-different-architectures.adoc (hunk 2 above)
Comment (agantony) [reply modules/installation-methods-for-different-architectures.adoc:28]: Done

File: modules/installation-methods-for-different-architectures.adoc (hunk 1 above)
Comment (agantony) [reply modules/installation-methods-for-different-architectures.adoc:23]: Done

**Pull request merged**